content/blog/posts/osu-story.rst

```diff
@@ -0,0 +1,11 @@
+OSL in the news!
+================
+:date: 2018-02-08
+:slug: osl-in-the-news
+:author: Jonathan Frederick
+:img: OSLWorkers2Adjusted.jpg
+:tag: featured-stories
+
+OSU has written a story about us! Check it out `here`_
```

Comment:
Can we have more than one sentence here? Maybe 2-3 which describes what the article is about?